Jenkinsfile

```diff
@@ -77,6 +60,8 @@ pipeline {
 					sh('kubectl --kubeconfig=${KUBERNETES_CREDS} -n dev apply -f banzai/tests/e2e-k8s.yaml')
 					// wait for the test pod to be running
 					sh('kubectl --kubeconfig=${KUBERNETES_CREDS} -n dev wait --for=condition=Ready --timeout=60m pod/banzai-e2e-test')
+					// sleep for an additional 30 and allow all containers to get into the proper state
+					sh('sleep 30s')
```

Comment:
Is this necessary? This should be taken care of in the wait command above. If not, we should make sense the containers are not set to ready until they can be used.

Reply:
Added readiness probes to the rabbitmq, redis, and celery containers to make sure we are ready for test invocation to avoid intermittent failures.